### PR TITLE
S.O.G. nickel steel

### PR DESCRIPTION
**Pull request by wersal454**
## What type of PR is this?
1. [ ] Bug
2. [ ] Change
3. [ ] Enhancement
4. [x] Miscellaneous

### What have you changed and why?
Information:
continuation of #185 cause I'm lazy and already wrote a code for it

Added plane from nickel steel mod in 2 variants:
bomber and transport

### Please verify the following.

1. [x] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?

1. [x] No
2. [ ] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps:
call QRFAIR and CASDIVE(or just CAS)
********************************************************
Notes:
Had to add commander fire code, same as gunner, doesn't break anything @stutpip123 and if plane doesn't have either gunner or commander it behaves normally.
Basically now you can add planes that have gunners or commanders controlling bombs

### CODE REVIEW

File: A3A/addons/core/functions/Base/fn_setPlaneLoadout.sqf
Comment (Silence-112) [inline A3A/addons/core/functions/Base/fn_setPlaneLoadout.sqf:1]: Ya know we have a hpp file for this right?

Comment (wersal454) [reply A3A/addons/core/functions/Base/fn_setPlaneLoadout.sqf:1]: Yeah I know, but I still don't understand why + I just got used to do it in sqf

Comment (Silence-112) [reply A3A/addons/core/functions/Base/fn_setPlaneLoadout.sqf:1]: Easier to read, rather than the spaghetti that is sqf

**Review by Silence-112 — APPROVED** [review]

File: A3A/addons/core/functions/Base/fn_setPlaneLoadout.sqf
Comment (wersal454) [reply A3A/addons/core/functions/Base/fn_setPlaneLoadout.sqf:1]: image
image
image

Comment (Silence-112) [reply A3A/addons/core/functions/Base/fn_setPlaneLoadout.sqf:1]: The other benefit is it not being in sqf, so it's harder to break the entire thing
Also it allows people who make extender mods to modify them if they wish, which you can't (easily) do with the sqf version

**Comment by Silence-112** [comment]
Fun, more merge conflicts

**Comment by wersal454** [comment]
> Fun, more merge conflicts

no more!
